packages/loader/container-loader/src/container.ts

```diff
@@ -1619,11 +1619,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             if (errorMsg !== undefined) {
                 const error = new GenericError(
```

Comment:
want to make this a datacorruption as part of #4663 while you are here?

Reply:
Ah yes of course!  I'll leave in the changes to `GenericError` even though they're not used now, since I went to the trouble of writing that test ;)